scripts/build-wasm.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2021 Adobe
+# All Rights Reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -e
+
+rm -rf build/wasm
+mkdir -p build/wasm
+
+CMAKE_ARGS=()
+
+# CMake-level configuration
+CMAKE_ARGS+=("-DCMAKE_BUILD_TYPE=Release")
+CMAKE_ARGS+=("-DXNNPACK_LIBRARY_TYPE=static")
+
```

Comment:
Unless you found that Ninja build doesn't work, please support building with Ninja when its installed in the system: https://github.com/google/XNNPACK/blob/master/scripts/build-local.sh#L19-L23
Building with Ninja is faster than with Make.